binding/Binding/Skottie/InvalidationController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace SkiaSharp.Skottie
+{
+	public unsafe class InvalidationController : SKObject
+	{
+		public InvalidationController ()
+			: this (IntPtr.Zero, true)
+		{
+			Handle = SkiaApi.sksg_invalidation_controller_new ();
+		}
+
+		internal InvalidationController (IntPtr handle, bool owns)
+			: base (handle, owns)
+		{
+		}
+
+		// SK_C_API void sksg_invalidation_controller_inval (sksg_invalidation_controller_t* instance, sk_rect_t* rect, sk_matrix_t* matrix);
+		public unsafe void Inval(SKRect rect, SKMatrix matrix)
```

Comment:
Can we use the full Invalidate here since we are not saving that much for the very non-.NET style.